### ISSUE TITLE
Create page link detail view = mouse-only

### ISSUE THREAD

**Issue by rebeccacremona** [harvard-lil/perma#2255]
You cannot presently toggle links to the detail view using your keyboard.

That means that the dropdown for moving links to new folders (the primary work-around if dragging+dropping is an issue) isn't actually exposed to people using keyboards.

Can be solved by the addition of an 'expand' button. See related https://github.com/harvard-lil/perma/issues/2173

**Comment by rebeccacremona** [comment]
Fixed!
